bin/check-snmp.rb

```diff
@@ -60,6 +60,11 @@ class CheckSNMP < Sensu::Plugin::Check::CLI
          description: 'Operator used to compare data with warning/critial values. Can be set to "le" (<=), "ge" (>=).',
          default: 'ge'
 
+  option :convert_timeticks,
+         short: '-T (true|false)',
+         description: 'Convert SNMP::TimeTick to Integer for comparisons',
+         default: 'false'
+
```

Comment:
it would be better convention to use a boolean here. Then it would look like: 

```ruby
option :convert_timeticks,
       short: '-T',
       long: '--convert-timeticks',
       description: 'Convert SNMP::TimeTick to Integer for comparisons',
       boolean: true,
       default: false
```

and you'd set `-T/--convert-timeticks` to enable that option.